sdk/keyvault/azure-security-keyvault-administration/src/test/java/com/azure/security/keyvault/administration/KeyVaultCredentialPolicyTest.java

```diff
@@ -182,9 +189,21 @@ public void onAuthorizeRequestNoScope() {
         assertFalse(onChallenge);
     }
 
+    @SyncAsyncTest
+    public void onAuthorizeRequestDifferentScope() {
+        KeyVaultCredentialPolicy policy = new KeyVaultCredentialPolicy(this.credential, true);
```

Comment:
Might also want a negative assertion test - that disabling verification works for an invalid resource.